src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -344,109 +342,81 @@ public Quaternion RotateUnitQuaternion(Quaternion unitQuaternion)
+                   && q.ImagZ.AlmostEqual(0);
         }
 
         /// <summary>
-        /// Negate this quaternion.
-        /// </summary>
-        public Quaternion Negate()
+        /// Inequality operator for two quaternions
+        /// </summary> 
+        public static bool operator !=(Quaternion q1, Quaternion q2)
         {
-            return new Quaternion(-_w, -_x, -_y, -_z, _abs, _norm, Math.PI - _arg);
+            return !(q1 == q2);
         }
-
         /// <summary>
-        /// Multiply a quaternion with this quaternion.
-        /// </summary>
-        public Quaternion Multiply(Quaternion q)
+        /// Inequality operator for quaternion and double
+        /// </summary> 
+        public static bool operator !=(Quaternion q1, double d)
         {
-            double ci = (+_x*q._w) + (_y*q._z) - (_z*q._y) + (_w*q._x);
-            double cj = (-_x*q._z) + (_y*q._w) + (_z*q._x) + (_w*q._y);
-            double ck = (+_x*q._y) - (_y*q._x) + (_z*q._w) + (_w*q._z);
-            double cr = (-_x*q._x) - (_y*q._y) - (_z*q._z) + (_w*q._w);
-            return new Quaternion(cr, ci, cj, ck);
+            return !(q1 == d);
         }
-
         /// <summary>
-        /// Multiply a floating point number to this quaternion.
-        /// </summary>
-        public Quaternion Multiply(double d)
+        /// Inequality operator for quaternion and double
+        /// </summary> 
+        public static bool operator !=(double d, Quaternion q1)
         {
-            return new Quaternion(d*_w, d*_x, d*_y, d*_z);
+            return !(q1 == d);
         }
-
         /// <summary>
-        /// Multiplies a Quaternion with the inverse of another
-        /// Quaternion (q*q<sup>-1</sup>). Note that for Quaternions
-        /// q*q<sup>-1</sup> is not the same then q<sup>-1</sup>*q,
-        /// because this will lead to a rotation in the other direction.
+        /// Convert a floating point number to a quaternion.
         /// </summary>
-        public Quaternion Divide(Quaternion q)
+        public static implicit operator Quaternion(double d)
         {
-            return Multiply(q.Inverse());
+            return new Quaternion(d, 0, 0, 0);
         }
-
         /// <summary>
-        /// Multiplies a Quaternion with the inverse of a real number.
+        /// Negate this quaternion.
         /// </summary>
-        /// <remarks>
-        /// Its also possible to cast a double to a Quaternion and make the division
-        /// afterward, but that would be more expensive.
-        /// </remarks>
-        public Quaternion Divide(double d)
+        public Quaternion Negate()
         {
-            return new Quaternion(_w/d, _x/d, _y/d, _z/d);
+            return new Quaternion(-_w, -_x, -_y, -_z);
         }
-
         /// <summary>
-        /// Inverts this quaternion.
+        /// Inverts this quaternion. Inversing Zero returns Zero
         /// </summary>
-        public Quaternion Inverse()
+        public Quaternion Inversed
```

Comment:
IMO this would better be a method as well (e.g. `Invert()`) instead of a property. Any thoughts?